packages/contracts-bedrock/test/L2OutputOracle.t.sol

```diff
@@ -416,21 +450,21 @@ contract L2OutputOracleUpgradeable_Test is L2OutputOracle_Initializer {
         assertEq(oracle.proposer(), proposer);
         assertEq(oracle.CHALLENGER(), oracleChallenger);
         assertEq(oracle.challenger(), oracleChallenger);
+        assertEq(address(oracle.systemConfig()), address(sysConf));
     }
 
     /// @dev Tests that the impl is created with the correct values.
     function test_initValuesOnImpl_succeeds() external {
-        assertEq(submissionInterval, oracleImpl.SUBMISSION_INTERVAL());
-        assertEq(l2BlockTime, oracleImpl.L2_BLOCK_TIME());
+        assertEq(oracle.SUBMISSION_INTERVAL(), submissionInterval);
+        assertEq(oracle.submissionInterval(), submissionInterval);
+        assertEq(oracle.L2_BLOCK_TIME(), l2BlockTime);
+        assertEq(oracle.l2BlockTime(), l2BlockTime);
 
         // The values that are set in the initialize function should be all
         // zero values in the implementation contract.
         assertEq(oracleImpl.startingBlockNumber(), 0);
         assertEq(oracleImpl.startingTimestamp(), 0);
-        assertEq(oracleImpl.PROPOSER(), address(0));
-        assertEq(oracleImpl.proposer(), address(0));
-        assertEq(oracleImpl.CHALLENGER(), address(0));
-        assertEq(oracleImpl.challenger(), address(0));
```

Comment:
Are we breaking the ABI by removing these getters? Wondering if any of our tooling uses them

Reply:
I haven't removed the getters, they are still tested on the proxy on lines 415-418. But they fail on the implementation since it depends on a call to the `systemConfig` address which is zero on the impl.